src/views/Help/help.scss

```diff
@@ -1,7 +1,36 @@
 @import '../../vars';
 
-strong {
-  font-size: 15px;
-  font-weight: bold;
-  margin-top: 20px;
+
+
+.help {
+  
+  strong {
+    font-weight: bold;
+    margin-top: 20px;
+  }
```

Comment:
Normally, we would need to check every instance of `<strong>` across the rest of the site to ensure this didn't break any other styling. However, I just discovered that this exact same styling is added at the top-level `app.scss` file (which is intended to be site-wide):
```
strong {
  font-size: 15px;
  font-weight: bold;
  margin-top: 20px;
}
```
So this particular styling is totally redundant, and should probably be removed.

Reply:
I see. I'm not sure I understood exactly before. Should we revert this until we can resolve that separate issue?

Reply:
Well, now that we know this is (intentionally) a site-wide style from `app.scss`, this particular repeat of it is irrelevant and never affects anything. So for this pull request, I think you can just delete it.

Reply:
great.